CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # Change log
 
+## 5.0.0 (???)
+
+* (troessner) Drop legacy code comment separator
+* (troessner) List all smells
+* (troessner) Drop Ruby 2.1 support
+* (mvz) Allow only detector names in configuration hash
+* (David Fisher) Change PrimaDonnaMethod to MissingSafeMethod
+* (mvz) Do not accept a class as parameter for reek_of
+* (troessner) Add yml extension to Reeks configuration file
+* (troessner) Move directory directives under a special key
+WIP * (troessner) Use strings not regexes in our configuration file
+WIP * (troessner) Add schema validation of our configuration
+WIP * (mvz) Update app configuration api
+WIP * (troessner) Fix Reek on CodeClimate
```

Comment:
Of course we need some short examples for most of the points above to not alienate users that upgrade.